### ISSUE TITLE
Invalid reference in deprecation notices regarding types (DefaultTypes)

### ISSUE THREAD

**Issue by Pnoexz** [doctrine/dbal#3717]
### Bug Report

|    Q        |   A
|------------ | ------
| BC Break    | no
| Version     | 2.10.0

#### Summary

In [this pull request](https://github.com/doctrine/dbal/pull/3356/files) a new deprecation around Types was introduced. It looks like the author of the PR initially used `DefaultTypes` for the name of the new class, but was later changed to Types, leaving references to the former in @see tags on the deprecation notices. [example](https://github.com/doctrine/dbal/blob/master/lib/Doctrine/DBAL/Types/Type.php#L20)

```
vendor/doctrine/dbal/lib/Doctrine/DBAL/Types/Type.php
21:/** @deprecated Use {@see DefaultTypes::BIGINT} instead. */
24:/** @deprecated Use {@see DefaultTypes::BINARY} instead. */
27:/** @deprecated Use {@see DefaultTypes::BLOB} instead. */
30:/** @deprecated Use {@see DefaultTypes::BOOLEAN} instead. */
33:/** @deprecated Use {@see DefaultTypes::DATE_MUTABLE} instead. */
36:/** @deprecated Use {@see DefaultTypes::DATE_IMMUTABLE} instead. */
39:/** @deprecated Use {@see DefaultTypes::DATEINTERVAL} instead. */
42:/** @deprecated Use {@see DefaultTypes::DATETIME_MUTABLE} instead. */
45:/** @deprecated Use {@see DefaultTypes::DATETIME_IMMUTABLE} instead. */
48:/** @deprecated Use {@see DefaultTypes::DATETIMETZ_MUTABLE} instead. */
51:/** @deprecated Use {@see DefaultTypes::DATETIMETZ_IMMUTABLE} instead. */
54:/** @deprecated Use {@see DefaultTypes::DECIMAL} instead. */
57:/** @deprecated Use {@see DefaultTypes::FLOAT} instead. */
60:/** @deprecated Use {@see DefaultTypes::GUID} instead. */
63:/** @deprecated Use {@see DefaultTypes::INTEGER} instead. */
66:/** @deprecated Use {@see DefaultTypes::JSON} instead. */
69:/** @deprecated Use {@see DefaultTypes::JSON_ARRAY} instead. */
72:/** @deprecated Use {@see DefaultTypes::OBJECT} instead. */
75:/** @deprecated Use {@see DefaultTypes::SIMPLE_ARRAY} instead. */
78:/** @deprecated Use {@see DefaultTypes::SMALLINT} instead. */
81:/** @deprecated Use {@see DefaultTypes::STRING} instead. */
84:/** @deprecated Use {@see DefaultTypes::ARRAY} instead. */
87:/** @deprecated Use {@see DefaultTypes::TEXT} instead. */
90:/** @deprecated Use {@see DefaultTypes::TIME_MUTABLE} instead. */
93:/** @deprecated Use {@see DefaultTypes::TIME_IMMUTABLE} instead. */

vendor/doctrine/dbal/lib/Doctrine/DBAL/Types/Types.php
37:/** @deprecated json_array type is deprecated, use {@see DefaultTypes::JSON} instead. */
```

**Comment by stof** [comment]
The class name is indeed Types, not DefaultTypes

**Comment by morozov** [comment]
FIxed by #3723.
